.changeset/nice-planets-roll.md

```diff
@@ -0,0 +1,6 @@
+---
+"@builder.io/sdk-qwik": patch
+---
+
+- Fix: Content editor hangs if not in the iframe viewport
+- Chore: Added test case to trigger event when content component is not in the viewport
```

Comment:
We usually only list user-facing changes in our release notes. Changes to testing and other infra can be ignored, as they do not impact the end-user.

```suggestion
Fix: Content editor hangs if not in the iframe viewport.
```